packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -45,14 +46,20 @@ export default function BookmarkFeedLayout({
         feedName: 'search-bookmarks',
         feedQueryKey: defaultKey.concat(searchQuery),
         query: SEARCH_BOOKMARKS_QUERY,
-        variables: { query: searchQuery },
+        variables: {
+          query: searchQuery,
+          supportedTypes: [PostType.Article, PostType.Share, PostType.Welcome],
```

Comment:
Suggestion can we add one export somewhere for supportedTypes?
It makes it way easier to maintain and less error prone?

Reply:
@rebelchris added it here 44bcbc9ed0e1ac6dc167d432d79b7ba35c97d36d, not 100% on naming though but I think just `supportedTypes` would not work because we do not support `Welcome` on regular feeds.

Reply:
Ah you are right, in either way we could have two maybe then?

Reply:
We already have `SUPPORTED_TYPES` that is just a string that we concat into queries as default.

Reply:
@rebelchris forgot to tag, if above is good I would merge because this PR is needed for pulse animation PR.

Reply:
Yep, all good :)